tests/table/test_init.py

```diff
@@ -793,6 +794,34 @@ def test_update_metadata_set_snapshot_ref(table_v2: Table) -> None:
     )
 
 
+def test_update_remove_snapshots(table_v2: Table) -> None:
+    update = RemoveSnapshotsUpdate(
+        snapshot_ids=[3051729675574597004],
+    )
+    new_metadata = update_table_metadata(table_v2.metadata, (update,))
+    assert len(new_metadata.snapshots) == 1
```

Comment:
if theres 1 snapshot, isnt that the current snapshot. if so, wouldnt it error?

Reply:
This is what's left after the removal, originally fixture has 2 snapshots